cmd/wallet-web/src/store/modules/options.js

```diff
@@ -49,7 +49,7 @@ let defaultAgentStartupOpts = {
   didAnchorOrigin: 'origin',
   sidetreeToken: '',
   hubAuthURL: 'https://localhost:8044',
-  staticAssetsUrl: '',
+  staticAssetsUrl: 'https://localhost:8091',
```

Comment:
default should be empty, if you need a URL for dev mode then there 'is dev mode' check somewhere in same file, put it there.

Reply:
done

Reply:
you can add it here https://github.com/trustbloc/wallet/blob/68d6fe530bacca4cdd09417fe9b74e5bb38b8681/cmd/wallet-web/src/store/modules/options.js#L151